packages/vscode-extension/src/startup/StatusBars.ts

```diff
@@ -0,0 +1,135 @@
+import * as vscode from 'vscode';
+import { FILES } from '../ApplicationConstants';
+import { commandsInfoMap } from '../commandsMap';
+import { STATUS_BARS } from '../service/TranslationKeys';
+import { VSTranslationService } from '../service/VSTranslationService';
+
+const translationService = new VSTranslationService();
+
+/**
+ * Create the status bar item used to display the active suitecloud project
+ *
+ * @returns {vscode.StatusBarItem} The suitecloud project status bar item
+ */
+export function createSuiteCloudProjectStatusBar(): vscode.StatusBarItem {
+	const suitecloudProjectStatusBar = vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Left, 10);
+	suitecloudProjectStatusBar.tooltip = translationService.getMessage(STATUS_BARS.SUITECLOUD_PROJECT.TOOLTIP);
+	suitecloudProjectStatusBar.hide();
+
+	return suitecloudProjectStatusBar;
+}
+
+/**
+ * Create the status bar item used to display the current authid used in the active suitecloud project
+ *
+ * @returns {vscode.StatusBarItem} The authID status bar item
+ */
+export function createAuthIDStatusBar(): vscode.StatusBarItem {
+	const authIDStatusBar = vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Left, 10);
+	authIDStatusBar.tooltip = translationService.getMessage(STATUS_BARS.AUTH_ID.TOOLTIP);
+	authIDStatusBar.command = commandsInfoMap.setupaccount.vscodeCommandId;
+	authIDStatusBar.hide();
+
+	return authIDStatusBar;
+}
+
+/**
+ * Update the active suitecloud project and used authID based on textEditor
+ *
+ * @param {vscode.TextEditor|undefined} textEditor
+ * @param {vscode.StatusBarItem} suitecloudProjectStatusBar
+ * @param {vscode.StatusBarItem} authIDStatusBar
+ */
+export async function updateStatusBars(
+	textEditor: vscode.TextEditor | undefined,
+	suitecloudProjectStatusBar: vscode.StatusBarItem,
+	authIDStatusBar: vscode.StatusBarItem
+) {
+	const activeTextEditor = textEditor;
```

Comment:
Maybe we can just rename the function parameter instead of re-assigning it

Reply:
Renamed some of the variables to only use param `textEditor` and not re-assign it.